nix/lib.nix

```diff
@@ -0,0 +1,188 @@
+lib: with lib; let
```

Comment:
Please don't use `with lib;`

Reply:
Sure, will use `inherit`. For some reason I remembered that `lib/generators.nix` uses `with lib;`, but it does not.

Reply:
Done.